pyro/infer/mcmc/mcmc.py

```diff
@@ -264,9 +266,21 @@ def __init__(self, kernel, num_samples, warmup_steps=None,
             self.sampler = _SingleSampler(kernel, num_samples, self.warmup_steps, disable_progbar)
         super(MCMC, self).__init__(num_chains=num_chains)
 
+    # TODO: Refactor so that the MCMC class directly has access to the
+    # trace generator and transforms needed to do this wrapping.
```

Comment:
Could you also add a TODO for mentioning about the params from kernel lie in unconstrained space?